src/SFML/Window/WindowBase.cpp

```diff
@@ -217,6 +217,23 @@ void WindowBase::setSize(const Vector2u& size)
     }
 }
 
+////////////////////////////////////////////////////////////
+void WindowBase::setMinimumSize(const std::optional<Vector2u>& minimumSize)
+{
+    if (m_impl)
+    {
+        m_impl->setMinimumSize(minimumSize);
+    }
+}
+
+////////////////////////////////////////////////////////////
+void WindowBase::setMaximumSize(const std::optional<Vector2u>& maximumSize)
+{
+    if (m_impl)
+    {
+        m_impl->setMaximumSize(maximumSize);
+    }
```

Comment:
```suggestion
    if (m_impl)
        m_impl->setMaximumSize(maximumSize);
```
Braces aren't used for similar things elsewhere in this file.